src/vm/exceptionhandling.cpp

```diff
@@ -7047,5 +7047,38 @@ void ExceptionTracker::ResetThreadAbortStatus(PTR_Thread pThread, CrawlFrame *pC
 }
 #endif //!DACCESS_COMPILE
 
+#if defined(_TARGET_ARM_) || defined(_TARGET_X86_)
```

Comment:
A nit - could you please move this function somewhere before the IsSafeToHandleHardwareException, mark the function as static and remove the function declaration from the exceptionhandling.h? It is not needed anywhere else, so there is no need to expose it.